src/openvslam/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ else()
         include_directories(${CXSPARSE_INCLUDE_DIRS})
     endif()
     # SuiteSparse for g2o
-    find_package(SuiteSparse)
+    find_package(SuiteSparse CONFIG)
```

Comment:
Is this change necessary?

Reply:
This is because of the missing cs.h file.

There is an issue on the g2o repo:
https://github.com/RainerKuemmerle/g2o/issues/182